src/main/java/thecount/ui/Reply.java

```diff
@@ -0,0 +1,24 @@
+package thecount.ui;
+
+/**
+ * Represents a reply message to be displayed to the user.
+ */
+public class Reply {
```

Comment:
Perhaps this is too detailed, I think you can combine all the replies into one class file 👍